cdk/bakery_stack.py

```diff
@@ -1,10 +1,211 @@
-from aws_cdk import core
+import os
+
+from aws_cdk import (
+    aws_ec2,
+    aws_ecr,
+    aws_ecs,
+    aws_ecs_patterns,
+    aws_iam,
+    aws_s3,
+    aws_secretsmanager,
+    core,
+)
 
 
 class BakeryStack(core.Stack):
     def __init__(
         self, scope: core.Construct, construct_id: str, identifier: str, **kwargs
     ) -> None:
         super().__init__(scope, construct_id, **kwargs)
+        bucket = aws_s3.Bucket(
+            self,
+            id=f"flow-storage-bucket-{identifier}",
+            auto_delete_objects=True,
+            removal_policy=core.RemovalPolicy.DESTROY,
```

Comment:
We probably want to add an issue/or address this here, but we're going to want to have a way of doing `RETAIN` if it's a 'production' deployment vs `DESTROY` in a ephemeral style deployment for development/testing

Reply:
Let's shelve this until we are moving towards production deployments.